tests/unit/transport/drivers/test_aiokafka.py

```diff
@@ -918,8 +918,8 @@ def test_start_coordinator_span(self, *, cthread):
     def test_close(self, *, cthread, _consumer):
         cthread._consumer = _consumer
         cthread.close()
-        _consumer.set_close.assert_called_once_with()
-        _consumer._coordinator.set_close.assert_called_once_with()
+        _consumer._closed is True
+        _consumer._coordinator.close.assert_called_once_with()
```

Comment:
Same with the private variables

Reply:
I think should be `assert _consumer._closed`